packages/xrpl/src/models/transactions/payment.ts

```diff
@@ -199,6 +213,10 @@ export function validatePayment(tx: Record<string, unknown>): void {
     throw new ValidationError('PaymentTransaction: InvoiceID must be a string')
   }
 
+  if (!validateDomainID(tx.DomainID)) {
```

Comment:
ditto here

Reply:
I've incorporated your suggestions in https://github.com/XRPLF/xrpl.js/commit/6bec2eb54e5e1051dd3759449484b9bed91fdf5f